builder/utils.go

```diff
@@ -110,3 +111,55 @@ func AggregateMax(col string) AggregateSymbleBuilder {
 func AggregateMin(col string) AggregateSymbleBuilder {
 	return agBuilder("min(" + col + ")")
 }
+
+// OmitEmpty is a helper function to clear where map zero value
+func OmitEmpty(where map[string]interface{}, omitKey []string) map[string]interface{} {
+	for _, key := range omitKey {
+		v, ok := where[key]
+		if !ok {
+			continue
+		}
+
+		if isZero(reflect.ValueOf(v)) {
+			delete(where, key)
+		}
+	}
+	return where
+}
+
+// isZero reports whether a value is a zero value
+// Including support: Bool, Array, String, Float32, Float64, Int, Int8, Int16, Int32, Int64, Uint, Uint8, Uint16, Uint32, Uint64, Uintptr
+// Map, Slice, Interface, Struct
+func isZero(v reflect.Value) bool {
+	switch v.Kind() {
+	case reflect.Bool:
+		return !v.Bool()
+	case reflect.Array, reflect.String:
+		return v.Len() == 0
+	case reflect.Float32, reflect.Float64:
+		return v.Float() == 0
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		return v.Int() == 0
+	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64, reflect.Uintptr:
+		return v.Uint() == 0
+	case reflect.Map, reflect.Slice, reflect.Interface:
```

Comment:
There's a bug: `make(map[int]int) != nil` but its length equals zero, it should be omitted! And `[]int{}` etc...

blow are some test cases:
```
		{
			map[string]interface{}{"ss": []int{}},
			[]string{"ss"},
			map[string]interface{}{},
		},
		{
			map[string]interface{}{"m1": make(map[string]string), "m2": map[string]string{"foo": "hi"}},
			[]string{"m1", "m2"},
			map[string]interface{}{"m2": map[string]string{"foo": "hi"}},
		},
```